src/sentry/static/sentry/app/components/group/pluginActions.tsx

```diff
@@ -2,25 +2,45 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Modal from 'react-bootstrap/lib/Modal';
 
+import {Client} from 'app/api';
 import withApi from 'app/utils/withApi';
 import {addSuccessMessage, addErrorMessage} from 'app/actionCreators/indicator';
 import NavTabs from 'app/components/navTabs';
 import plugins from 'app/plugins';
-import {t} from 'app/locale';
+import {t, tct} from 'app/locale';
 import SentryTypes from 'app/sentryTypes';
 import IssueSyncListElement from 'app/components/issueSyncListElement';
 import withOrganization from 'app/utils/withOrganization';
+import {Organization, Group, Project, Plugin} from 'app/types';
 
-class PluginActions extends React.Component {
-  static propTypes = {
+type Props = {
+  api: Client;
+  group: Group;
+  organization: Organization;
+  project: Project;
+  plugin: Plugin & {
+    // issue serializer adds more fields
+    title: 'string';
```

Comment:
Can we add a todo note here that this shouldn't be needed as we think `name` should always exist, but it's here for compatibility reasons? I wonder if it'd be worthwhile to use `trackDeprecated` to see if this is being hit at all (`name` being empty) and just removing it if so.

```suggestion
    title: string;
```